reflex.yaml

```diff
@@ -32,13 +32,15 @@ pipeline:
     with:
       repository: https://github.com/reflex-dev/reflex
       tag: v${{package.version}}
-      expected-commit: ecf222d4b056c1237c1e42477805200d6c6bb492
+      expected-commit: 2b17a320d41a1869e680abb7d4957e9fb3cf36bc
 
   - runs: |
       #Fix CVEs
       poetry add "gunicorn==22.0.0"
       poetry add "certifi==2024.07.04"
       poetry add "idna==3.7"
+      poetry add "urllib3==1.26.19"
+      poetry add "setuptools==70.0.0"
```

Comment:
Looks like this didn't take affect:

```
└── 📄 /usr/lib/python3.12/site-packages/virtualenv/seed/wheels/embed/setuptools-68.0.0-py3-none-any.whl
        📦 setuptools 68.0.0 (python)
            High CVE-2024-6345 GHSA-cx63-2mw6-8hw5 fixed in 70.0.0
```

Though I don't see anywhere else we're using this as a runtime dependency in any other packages or images. If the version bump isn't working here we should remove it  - we can still merge the package update

Reply:
Thanks, reverted my previous commit.